Build failure with case-insensitive-1.2.1.0

I made a revision https://hackage.haskell.org/package/xml-lens-0.1.6.3/revisions/

```
Text/XML/Lens.hs:154:47: error:
    Ambiguous occurrence ‘traverse’
    It could refer to either ‘Control.Lens.traverse’,
                             imported from ‘Control.Lens’ at Text/XML/Lens.hs:61:1-19
                             (and originally defined in ‘Data.Traversable’)
                          or ‘CI.traverse’,
                             imported from ‘Data.CaseInsensitive’ at Text/XML/Lens.hs:65:1-33
                             (and originally defined in ‘case-insensitive-1.2.1.0:Data.CaseInsensitive.Internal’)
```

I'd recommend using `import qualified Data.CaseInsensitive as CI`

--- a/Text/XML/Lens.hs
+++ b/Text/XML/Lens.hs
@@ -62,7 +62,7 @@ import Control.Lens
 import Data.Text (Text)
 import Data.Map (Map)
 import Control.Applicative
-import Data.CaseInsensitive as CI
+import qualified Data.CaseInsensitive as CI
 
 infixr 9 ./
 

--- a/xml-lens.cabal
+++ b/xml-lens.cabal
@@ -21,4 +21,4 @@ library
   default-language:   Haskell2010
   ghc-options: -Wall
   exposed-modules:     Text.XML.Lens
-  build-depends:       base ==4.*, lens >= 4.0 && < 5, containers >= 0.4.0 && < 0.7, text >= 0.7 && <2, xml-conduit >= 1.1 && < 1.4, case-insensitive
+  build-depends:       base ==4.*, lens >= 4.0 && < 5, containers >= 0.4.0 && < 0.7, text >= 0.7 && <2, xml-conduit >= 1.1 && < 1.9, case-insensitive

